src/SwedbankPay.Sdk/PaymentInstruments/Swish/SwishPaymentOptionsObject.cs

```diff
@@ -1,13 +1,13 @@
-﻿namespace SwedbankPay.Sdk.Payments.SwishPayments
-{
-    public class SwishPaymentOptionsObject
-    {
-        public SwishPaymentOptionsObject(bool ecomOnlyEnabled = false)
-        {
-            EcomOnlyEnabled = ecomOnlyEnabled;
-        }
-
-
-        public bool EcomOnlyEnabled { get; }
-    }
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Swish
+{
+    public class SwishPaymentOptionsObject
+    {
+        public SwishPaymentOptionsObject(bool ecomOnlyEnabled = false)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)